js/tic_tac_toe.js

```diff
@@ -1,2 +1,140 @@
-// Your code here
+
 // Use the jQuery
+var App = (function() {
+
+  var $boxCell, $box1, $box2, $box3, $box4, $box5, $box6, $box7, $box8, $box9;
+  var x = 1;
+
+  var init = function() {
+
+    $boxCell = $('.box_cell');
+    $('tr').on('click', '.box_cell', App.changeColor );
+    $('tr').on('click', '.box_cell', App.winner );
+
+  }
+
+  var changeColor = function(){
+    if($(this).attr('class') != "box_cell red" && $(this).attr('class') != "box_cell black"){
+      if (x%2 != 0) {
+        $(this).css('background-color', 'red');
+        $(this).addClass("red");
+        x += 1;
+      } else if (x%2 == 0) {
+        $(this).css('background-color', 'black');
+        $(this).addClass("black");
+        x += 1;
+      }
+    }
+  };
+
+  var winner = function(){
+
+    $box1 = $('#box_0_0');
+    $box2 = $('#box_0_1');
+    $box3 = $('#box_0_2');
+
+    $box4 = $('#box_1_0');
+    $box5 = $('#box_1_1');
+    $box6 = $('#box_1_2');
+
+    $box7 = $('#box_2_0');
+    $box8 = $('#box_2_1');
+    $box9 = $('#box_2_2');
+
+    //row 1
```

Comment:
Obviously this is a little long, and you'll see how you can make it shorter- but getting it done is more important than elegance, as that can be fixed later.